updating github action config for windows test (#387)

bfitzpat opened the pull request:
Signed-off-by: Brian Fitzpatrick <bfitzpat@redhat.com>

apupier commented:
there is no issue number to at least have a trace that the test should be reactivated later on

bfitzpat commented:
> there is no issue number to at least have a trace that the test should be reactivated later on

Updated the JIRA here - https://issues.redhat.com/browse/FUSETOOLS2-895 and added a follow-up JIRA for the flaky tests to get them re-activated https://issues.redhat.com/browse/FUSETOOLS2-969
